### PR TITLE
PURCHASE-2168: Adjusting BNMO order form's "Shipping" breadcrumb

### PR DESCRIPTION
**Pull request by The-Beez-Kneez**
This is to resolve the bug that had no padding on BNMO order form's "Shipping" breadcrumb. Also added in additional instructions for artsy/force set up.

This resolves the following JIRA ticket:
PURCHASE-2168

Fixed behavior:
Screen Shot 2021-02-04 at 2 03 25 PM

### CODE REVIEW

File: src/v2/Apps/Order/Components/OrderStepper.tsx [hunk 1]
```diff
@@ -14,6 +16,10 @@ export const offerFlowSteps = typedArray(
 export const buyNowFlowSteps = typedArray("Shipping", "Payment", "Review")
 export const counterofferFlowSteps = typedArray("Respond", "Review")
 
+const StyledContainer = styled.div`
+  padding-left: 20px;
+`
```
Comment (damassi) [inline src/v2/Apps/Order/Components/OrderStepper.tsx:21]: (thought) Since we have Palette (our design system), we can just [use our `Box` component](https://palette.artsy.net/elements/layout/box/) and avoid intermediate wrappers completely. 

(In general, we should always use Palette components, be they `Box` or `Flex`, and avoid things like `divs` since they lack all of the functionality that our [Theme file](https://github.com/artsy/palette/blob/master/packages/palette/src/Theme.tsx) provides.)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -98,11 +98,15 @@ that behavior even for subdomains (such as local.artsy.net).
...
-3. Update your `.env` file's `APP_URL` and `RECAPTCHA_KEY`:
+3. Update your `.env` file's `APP_URL`, `RECAPTCHA_KEY`, `ARTSY_ID`, and `ARTSY_SECRET`:
+   Note: You can pull the majority of the environment variables from staging config, running:
+   `hokusai staging env get`
 
 ```
 APP_URL=http://local.artsy.net:5000
 RECAPTCHA_KEY=REPLACE_ME
+ARTSY_ID=REPLACE_ME
+ARTSY_SECRET=REPLACE_ME
```
Comment (iskounen) [inline CONTRIBUTING.md:109]: 💜

Comment (iskounen) [inline CONTRIBUTING.md:109]: Thanks for updating the docs! 💜

File: src/v2/Apps/Order/Components/OrderStepper.tsx (hunk 1 above)
Comment (jpotts244) [reply src/v2/Apps/Order/Components/OrderStepper.tsx:21]: Good call @damassi! I definitely suggested a `div` here :-)

Comment (The-Beez-Kneez) [reply src/v2/Apps/Order/Components/OrderStepper.tsx:21]: Hey @damassi, I got a brief overview on Palette from @lilyfromseattle. I was unsure what elements of palette to utilize here, as we had styled components in other locations in the repo. Is there a general rule on when we style and when we utilize an element from palette?

Comment (The-Beez-Kneez) [reply src/v2/Apps/Order/Components/OrderStepper.tsx]: Also updated the code, using `Box` now!

File: CONTRIBUTING.md (hunk 2 above)
Comment (The-Beez-Kneez) [reply CONTRIBUTING.md:109]: No worries!

File: src/v2/Apps/Order/Components/OrderStepper.tsx (hunk 1 above)
Comment (damassi) [reply src/v2/Apps/Order/Components/OrderStepper.tsx]: > Is there a general rule on when we style and when we utilize an element from palette

Yup! For example: 

```tsx
const Wrapper = styled(Box)`
  background-color: green;
`

const Foo = () => {
  return <Wrapper p={2} my={4}>...</Wrapper>
}
```

Because we've augmented a palette component, we can tap into our theme functionality and, for example, [use `p={2}`](https://github.com/artsy/palette/blob/master/packages/palette/src/Theme.tsx#L178-L179) to represent a `20px` padding, versus having to write it out in css longhand (and later, if the look and feel of artsy changes changes and everywhere we use 20px should actually be 30px, we can update it in our palette theme one time, versus doing find and replace across conventional CSS code). 

Basically, because of Palette we get the incredibly great joy of never really having to write vanilla CSS, because most everything is already covered by using the props that our palette components provide. If yr working on a component and notice that you've written a lot of custom, conventional CSS, its probably an indicator that one could do it more efficiently, and to reach out in the #practice-web or #dev-help channel for a second set of eyes!

Comment (The-Beez-Kneez) [reply src/v2/Apps/Order/Components/OrderStepper.tsx]: Awesome, thanks for the explanation!

**Review by damassi — APPROVED** [review]
LGTM! 👍
